### PR TITLE
Add predicting start work time based on workedTime. Add element-ui into white list

### PR DESCRIPTION
**Pull request by kanapka94**

### CODE REVIEW

File: src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue [hunk 1]
```diff
@@ -58,6 +69,24 @@ export default {
     closeDialog () {
       this.issue = null
       this.dialogVisible = false
+    },
+    handleBlur (event) {
+      const inputValue = event.target.value
+      const days = +this.removeLastCharacter(/([0-9]{1,2}d)+/ig.exec(inputValue))
+      const hours = +this.removeLastCharacter(/([0-9]{1,2}h)+/ig.exec(inputValue))
+      const minutes = +this.removeLastCharacter(/([0-9]{1,2}m)+/ig.exec(inputValue))
```
Comment (Argeento) [inline src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue]: Suggestion:
```javascript
const getNumberBeforeLetter = letter => {
  const regex = new RegExp(`(\\d+)${letter}`, 'i')
  return +(regex.exec(inputValue) || [0, 0])[1]
}

const [days, hours, minutes] = ['d', 'h', 'm'].map(getNumberBeforeLetter)
```

File: src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue [hunk 2]
```diff
@@ -58,6 +69,24 @@ export default {
...
+
+      const startDate = moment().subtract({
+        days,
+        hours,
+        minutes
+      }).format('YYYY-MM-DD HH:mm:ss')
+      if (!this.form.worklog.started) {
+        this.$set(this.form.worklog, 'started', new Date(startDate))
```
Comment (Argeento) [inline src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue]: Use [`toDate`](http://momentjs.com/docs/#/displaying/as-javascript-date/) istead of `format` method.

Comment (kanapka94) [reply src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue]: Thanks ! :)

File: src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue (hunk 1 above)
Comment (kanapka94) [reply src/renderer/components/dashboardPage/issues/contextMenu/actions/logWorkDialog/index.vue]: Thank you :)

**Review by Argeento — APPROVED** [review]

**Pull request merged**